lego/settings/lego.py

```diff
@@ -22,7 +22,7 @@
 PENALTY_IGNORE_SUMMER = ((6, 1), (8, 15))
 PENALTY_IGNORE_WINTER = ((11, 18), (1, 10))
 
-BEDKOM_BOOKING_PERIOD = ((10, 2), (10, 19))
+BEDKOM_BOOKING_PERIOD = ((1, 1), (12, 30))
```

Comment:
Desember has 31 days, not 30. I assume nobody sends company interests on new years eve, but it's very random to have auto reply on every day of the year except for one...